widgetssdk/src/main/java/com/glia/widgets/chat/StatefulChatInputBackground.kt

```diff
@@ -0,0 +1,62 @@
+package com.glia.widgets.chat
+
+import android.content.Context
+import android.graphics.Color
+import android.util.AttributeSet
+import android.view.View
+import androidx.annotation.ColorInt
+import com.glia.widgets.R
+import com.glia.widgets.helper.getAttr
+import com.glia.widgets.helper.getColorCompat
+import com.glia.widgets.messagecenter.SimpleStatefulWidgetAdapter
+import com.glia.widgets.messagecenter.StatefulWidgetAdapter
+import com.glia.widgets.messagecenter.StatefulWidgetAdapterCallback
+import com.glia.widgets.view.unifiedui.applyLayerTheme
+import com.glia.widgets.view.unifiedui.theme.base.ColorTheme
+import com.glia.widgets.view.unifiedui.theme.base.LayerTheme
+
+internal class StatefulChatInputBackground @JvmOverloads constructor(context: Context, attrs: AttributeSet? = null) : View(context, attrs) {
+
+    private val adapterCallback: StatefulWidgetAdapterCallback<State, LayerTheme> =
+        StatefulWidgetAdapterCallback { applyLayerTheme(it) }
+
+    private val statefulWidgetAdapter: StatefulWidgetAdapter<State, LayerTheme> by lazy {
+        SimpleStatefulWidgetAdapter(State.ENABLED, adapterCallback)
+    }
+
+    @get:ColorInt
+    private val gliaSystemAgentBubbleColor: Int by lazy {
+        getColorCompat(getAttr(R.attr.gliaSystemAgentBubbleColor, R.color.glia_neutral_color))
+    }
```

Comment:
Could you maybe make `gliaSystemAgentBubbleColor` as a `val disabledColor` value assign at class `init { }`. I mean this is a question of preference so it is up to you if you want to leave it as is.

Reply:
fixed - eccc1faaebbf293da3ecb4927c4fdf3a78aac252